packages/records/lib/db/migrations/20240327151027_create_records_table.ts

```diff
@@ -0,0 +1,104 @@
+import type { Knex } from 'knex';
+
+const TABLE = 'records';
+const PARTITION_COUNT = 256;
+
+function partitionTable(i: number): string {
+    return `${TABLE}_p${i}`;
+}
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.transaction(async (trx) => {
+        // TABLE
+        await trx.raw(`
+            CREATE TABLE "${TABLE}" (
+            id uuid NOT NULL,
+            external_id character varying(255) NOT NULL,
+            json jsonb,
+            data_hash character varying(255) NOT NULL,
+            connection_id integer NOT NULL,
+            model character varying(255) NOT NULL,
+            created_at timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
+            updated_at timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
+            deleted_at timestamp with time zone,
```

Comment:
Not sure if it's a good idea but can we take this opportunity to remove the timezone from this table?

Reply:
imo it's too complicated to handle date with timezone and provides no advantages, no big monstrous bug but some stuff will slightly differ from time to time which is inconvenient. 
e.g: If the script is running during Daylight Saving Time change, if the scripts are running in different timezones etc. 
Unless I'm missing something it's at the very least not useful

Reply:
since I am not sure if it is a good idea or not I am gonna keep it like it is. Timezones is a tricky topic and I am not sure there is one perfect solution